README.md

```diff
@@ -1,47 +1,19 @@
+**How do we accelerate?** A team can use our resources as guidance, reference, fork, or as dependencies.
 
-#### FAST JSS manager
-A React higher-order component (HOC) library for managing component JSS stylesheets. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-jss-manager-react)
+## Status
+[![Build Status](https://travis-ci.org/Microsoft/fast-dna.svg?branch=master)](https://travis-ci.org/Microsoft/fast-dna)
 
-#### FAST React viewer
-A self contained React component which shows content in an iframe. This can be used to as a method for previewing a React component(s) or an entire page. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-viewer)
 
+## Packages
+[Lerna](https://github.com/lerna/lerna) in [independent mode](https://github.com/lerna/lerna#independent-mode---independent) is used to organize and manage this monorepository and contains utilities each of which are published to NPM. Users can choose one package, some packages, or all of them together as an accelerator to quickly construct enterprise experiences. Highly tuned production code where the hard work has been done already.
 
-### General packages
-#### FAST animations
-An animation library that simplifies interactions and animations using the Web Animation API. Provides a convenient abstraction layer over the Web Animation API while still allowing full access to it. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-animation)
-
-#### FAST browser extensions
-An extension for testing real-world localization, theming, and other production type scenarios in web development. Originally devised as a way to visualize prototypes across many different user scenarios. For example, view a website in one of the 194 different localized settings. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-browser-extentions)
-
-#### FAST component contracts
-This package provides the TypeScript typings for all FAST components class-name contracts. These contracts enable strict typing of JSS stylesheets and each component's expectation of which class-names will be made available (and under which keys those class-names reside) to the component at runtime. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-components-class-name-contracts)
-
-#### FAST permutator
-Creates all possible data permutations from a given JSON Schema, allowing for deep testing or display of complex data sets. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-permutator)
-
-#### FAST tslint rules
-Provides [tslint](https://github.com/palantir/tslint) rules for all FAST projects. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-tslint-rules)
+[continue reading...](https://github.com/Microsoft/fast-dna/wiki/Packages)
```

Comment:
"Learn more about the FAST-DNA packages" or something similar. That would indicate to me that I'm going to a new page. The above makes me think that a section may expand